package/jest.config.js

```diff
@@ -1,13 +1,13 @@
 /* global require */
-// eslint-disable-next-line no-undef
+
 module.exports = {
   globalSetup: './jest-global-setup.js',
   moduleNameMapper: {
     'mock-builders(.*)$': '<rootDir>/src/mock-builders$1',
   },
-  preset: '@testing-library/react-native',
+  preset: 'react-native',
   setupFiles: [
-    './src/mock-builders/native/react-native-mock.js',
+    // './src/mock-builders/native/react-native-mock.js',
```

Comment:
Is this needed?

Reply:
No, it's no longer needed - the file it mocks has been moved elsewhere (and everything is working fine without it as well). I can remove it fully

Reply:
Scratch that, I updated the mock and brought it back just in case.